src/jshim/jshim.cc

```
@@ -691,7 +699,7 @@ void * instantiateJClassAsCXXClass(jobject obj)
    * Signature: (Lcom/opengamma/longdog/datacontainers/OGNumeric;)Lcom/opengamma/longdog/datacontainers/OGNumeric;
    */
   JNIEXPORT jobject JNICALL Java_com_opengamma_longdog_materialisers_Materialisers_materialise
-  (JNIEnv * env, jclass clazz, jobject obj)
+  (JNIEnv __attribute__ ((unused)) *env, jclass __attribute__ ((unused)) clazz, jobject obj)
```

It was to silence the errors, yes. I can add the SUPPRESS_UNUSED macro, or **attribute** ((unused)) can just be shortened to **unused** - which would you find preferable?